packages/share_plus_linux/lib/share_plus_linux.dart

```diff
@@ -14,9 +14,22 @@ class ShareLinux extends SharePlatform {
     String text, {
     String? subject,
     Rect? sharePositionOrigin,
-  }) {
-    final uri = Uri.encodeFull('mailto:?subject=$subject&body=$text');
-    return launch(uri);
+  }) async {
+    final queryParameters = {
+      if (subject != null) 'subject': subject,
+      'body': text,
+    };
+
+    final uri = Uri(
+      scheme: 'mailto',
+      queryParameters: queryParameters,
```

Comment:
very nice!!

Reply:
FYI, this is actually wrong due to a Dart bug (which is why I didn't suggest this format when I filed the issue). See https://github.com/dart-lang/sdk/issues/43838, especially https://github.com/dart-lang/sdk/issues/43838#issuecomment-823551891.

This will fix the issue I filed, but may cause spaces to be converted to `+` depending on the handler.

Reply:
That's unfortunate 😅 Are there more gotchas which I should be aware of before attempting another fix?

Reply:
None that I know of, no. I would suggest including spaces in the values in your unit tests on the next round, so that tests will prevent someone accidentally falling into this trap again in the future when maintaining these plugins (although hopefully the issue will be fixed in Dart itself at some point).